api/controller.py

```diff
@@ -1078,10 +1177,15 @@ def protocol(self):
 
 class AnalyticsController(CirculationManagerController):
 
-    def track_event(self, data_source, identifier_type, identifier, event_type):
+    def track_event(self, identifier_type, identifier, event_type):
+        # TODO: It usually doesn't matter, but there should be
+        # a way to distinguish between different LicensePools for the
+        # same book.
```

Comment:
We probably need to start exposing the URL for this in OPDS. I think the mobile apps currently construct it by replacing part of the OPDS entry's URL.